### PR TITLE
Duniverse v1.4.0

### PR DESCRIPTION
**Pull request by maiste**
v1.4.0 2022-02-17 La Forclaz (VS)
---------------------------------

* Change the `js_of_ocaml` strategy for `Mtime_clock`'s JavaScript
  implementation. Primitives of `mtime.clock.os` are now implemented
  in pure JavaScript and linked by `js_of_ocaml`.  This means that the
  `mtime.clock.jsoo` library no longer exists, simply link against
  `mtime.clock.os` instead. Thanks to Hugo Heuzard for suggesting and
  implementing this.

* Add `Mtime.{min,max}_stamp`.
* Add durations `Mtime.Span.{ns,us,ms,s,min,hour,day,year}` and
  the `Mtime.Span.(*)` operator (#28).
* Deprecate `Mtime.s_to_*` and `Mtime.*_to_s` floating point constants (#28).
* Require OCaml >= 4.08.
* Allow compiling with MSVC compiler. Thanks to Jonah Beckford for the patch.

### CODE REVIEW

File: mtime.opam [hunk 1]
```diff
@@ -19,14 +19,11 @@ maintainer: "Daniel Bünzli <daniel.buenzl i@erratique.ch>"
...
 depends: [
+  "dune"
   "ocaml" {>= "4.08.0"}
-  "ocamlfind" {build}
-  "ocamlbuild" {build & != "0.9.0"}
-  "topkg" {build & >= "1.0.3"}
 ]
-build: ["ocaml" "pkg/pkg.ml" "build" "--dev-pkg" "%{dev}%"]
-dev-repo: "git+https://erratique.ch/repos/mtime.git"
+tags: [ "time" "posix" "system" "org:erratique" ]
```
Comment (Leonidas-from-XIV) [inline mtime.opam:28]: It seems to be missing the `build` line entirely?

**Review by Leonidas-from-XIV — CHANGES_REQUESTED** [review]
Apart from the missing `build` line, I think this is very nice. Thanks for contributing the port!

**Review by Leonidas-from-XIV — APPROVED** [review]
Looks good! :+1:

**Comment by patricoferris** [comment]
I might be mistaken, but I think `1.4.0` may need #2 for it to work seamlessly in the browser :))

**Pull request merged**